softwerkskammer/lib/wiki/eventsToObject.js

```diff
@@ -1,33 +1,33 @@
 'use strict';
 
-var _ = require('lodash');
-var moment = require('moment-timezone');
+const moment = require('moment-timezone');
+const misc = require('../commons/misc');
```

Comment:
find ich trotzdem nicht gut, weil es unser ungeschriebenes Gesetz bricht, dass wir alles in den Beans ersetzen können und das dann überall im Code ersetzt ist.
Und Du weißt, es wird jemand kommen und genau diesen Fall sehen und kopieren, und dann...